bin/lcov

```diff
@@ -517,6 +520,8 @@ Operation:
   -l, --list FILE                 List contents of tracefile FILE
       --diff FILE DIFF            Transform tracefile FILE according to DIFF
       --summary FILE              Show summary coverage data for tracefiles
+      --fail-under-lines MIN      Exit with a status of 1 if the total line
+                                  coverage is less than MIN (summary option).
```

Comment:
Move this text down to the section named "Options". "Operations" are those options that specify the actual lcov operation (add, capture, etc.).

Reply:
Done.